openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedTab/ActivityFeedTab.component.tsx

```diff
@@ -463,6 +561,52 @@ export const ActivityFeedTab = ({
             )}
           </div>
         )}
+        {isAnnouncementActiveTab && (
+          <div className="d-flex gap-4 p-sm p-x-lg activity-feed-task justify-between">
+            <div className="d-flex gap-4">
+              {getElementWithCountLoader(
+                <Typography.Text
+                  className={classNames(
+                    'cursor-pointer p-l-xss d-flex items-center',
+                    {
+                      'font-medium':
+                        announcementFilter === AnnoucementStatus.Active,
+                    }
+                  )}
+                  data-testid="active-announcements"
+                  onClick={() => {
+                    handleUpdateAnnouncementFilter(AnnoucementStatus.Active);
+                    setActiveThread();
```

Comment:
can we use this callback in the separate function, as they won't re-construct them, after every page render

Reply:
Extracted this into a separate function